freebsd/sys/cdefs.h

```diff
@@ -276,7 +276,7 @@
 #endif
 
 #if !defined(__cplusplus) && !__has_extension(c_atomic) && \
-    !__has_extension(cxx_atomic)
+    !__has_extension(cxx_atomic) && !__GNUC_PREREQ__(4, 7)
```

Comment:
Shoud use `&& !__GNUC_PREREQ__(4, 7)`, see https://gcc.gnu.org/gcc-4.9/changes.html

Reply:
Great finding, seems correct that 4.9 is the boundary